Resources/Private/Form/Password.html

```diff
@@ -1,7 +1,8 @@
+{namespace form=Neos\Form\ViewHelpers}
 <f:layout name="Neos.Form:Field" />
 <f:section name="field">
 	<f:form.password property="{element.identifier}" id="{element.uniqueIdentifier}" class="{element.properties.elementClassAttribute}" errorClass="{element.properties.elementErrorClassAttribute}" />
 	<f:if condition="{element.properties.passwordDescription}">
-		<span class="help-block">{element.properties.passwordDescription -> f:translate(id: 'forms.elements.{element.identifier}.passwordDescription', package: '{element.renderingOptions.translationPackage}')}</span>
+		<span class="help-block">{element -> form:translateElementProperty(property: 'passwordDescription')}</span>
```

Comment:
Yes it does, see https://github.com/neos/form/blob/master/Classes/ViewHelpers/TranslateElementPropertyViewHelper.php#L68